src/ui/options/components/accounts.tsx

```diff
@@ -212,7 +213,14 @@ function Properties(props: { scrobbler: Scrobbler }) {
 											data = {};
 										}
 										data[typedKey] = e.currentTarget.value;
-										scrobbler.applyUserProperties(data);
+										scrobbler
+											.applyUserProperties(data)
+											.then(() => {
+												sendContentMessage({
+													type: 'updateScrobblerProperties',
+													payload: void 0,
```

Comment:
What's the significance of `void 0`? So we can send a message without a payload?

Reply:
Its the same as `undefined`, old habit. Can change if you want.

Reply:
It's fine :+1: